media-api/app/lib/querysyntax/QuerySyntax.scala

```diff
@@ -24,7 +24,7 @@ class QuerySyntax(val input: ParserInput) extends Parser with ImageFields {
   def ScopedMatch = rule { MatchField ~ ':' ~ MatchValue }
   def HashMatch = rule      { '#' ~ MatchValue      ~> (label      => Match(SingleField(getFieldPath("labels")), label)) }
   // All hierarchy values are lowercase as specified in the analyser
-  def CollectionRule = rule { '~' ~ ExactMatchValue ~> (collection => Match(HierarchyField(getFieldPath("pathHierarchy"), collection.string.toLowerCase), collection)) }
+  def CollectionRule = rule { '~' ~ ExactMatchValue ~> (collection => Match(HierarchyField, Phrase(collection.string.toLowerCase))) }
```

Comment:
This is a simplification to the code. Given a HierarchyField will only ever be used for collection queries (i.e. those that pass the collection rule), then we do not need to set a field path "pathHierarchy" and pass that through as we can imply it when we match on a HierarchyField [here]( https://github.com/guardian/grid/pull/1712/files#diff-690e3390cd34f6bbe4c278bd7348d7a2R37).